src/main/java/util/Helpers.java

```diff
@@ -20,8 +20,8 @@ public static void selectValueFromDropdownAppForm(WebDriver driver, By locatorDr
         String innerComp;
         driver.findElement(locatorDropdown).click();
         List<WebElement> businessNatureDropdown = driver.findElements(locatorList);
-        for (int i = 0; i < businessNatureDropdown.size(); i++) {
-            element = businessNatureDropdown.get(i);
+        for (WebElement aBusinessNatureDropdown : businessNatureDropdown) {
+            element = aBusinessNatureDropdown;
```

Comment:
no need in this
 element = aBusinessNatureDropdown; 
work with aBusinessNatureDropdown directly